addons/account_payment/wizards/payment_refund_wizard.py

```diff
@@ -58,8 +58,9 @@ def _compute_amount_to_refund(self):
     @api.depends('transaction_id.provider_id', 'transaction_id.payment_method_id')
     def _compute_support_refund(self):
         for wizard in self:
-            p_support_refund = wizard.transaction_id.provider_id.support_refund
-            pm = wizard.transaction_id.payment_method_id
+            tx_sudo = wizard.transaction_id.sudo()  # needed for users without access to the provider
```

Comment:
In cases like this, a comment is as good as a test, so no need to add one for that ;)